lib/ui/hooks.dart

```diff
@@ -4,6 +4,28 @@
 
 part of dart.ui;
 
+@pragma('vm:entry-point')
+void _updateDisplays(
+  List<int> ids,
+  List<double> sizes,
+  List<double> devicePixelRatios,
+  List<double> refreshRates,
+) {
```

Comment:
Consider adding an assert that all parameters' lengths are expected

Reply:
Done